distro/docker-compose/docker-compose.apicurio.yml

```diff
@@ -100,5 +100,6 @@ services:
     volumes:
       - ./config/keycloak/apicurio-realm.json:/opt/keycloak/data/import/apicurio-realm.json
       - ./config/keycloak/microcks-realm.json:/opt/keycloak/data/import/microcks-realm.json
-    network_mode: host
```

Comment:
why is this a problem but running the other components in host mode is ok?

Reply:
Hi Carles, thank you for reviewing. Because the 8080 port on the local machine has been occupied by the UI, if I'm not mistaken. Could you provide some pointers for better solutions?